apps/sla/models.py

```diff
@@ -29,6 +30,9 @@ class EndingTypes(models.TextChoices):
         # releases and this is the best naming I was able to come up with for this range
         NO_WEEK_ENDING = "no week ending"
 
+        # this means skip shutdown idle days from December 24th to January 2th
+        NO_SHUTDOWN = "no shutdown"
```

Comment:
This is nothing major but if you have no reason not to do it we should keep the alphabetic sorting of the options.